stage1/usr_from_kvm/usr_from_kvm.mk

```diff
@@ -1,10 +1,23 @@
 $(call setup-stamp-file,UFK_CBU_STAMP,cbu)
 $(call setup-tmp-dir,UFK_TMPDIR)
 
+HV_BUILD_FILE :=
```

Comment:
This may break `make clean` in a certain scenario (configure with kvm-qemu flavor, make, configure with kvm-lkvm flavor, make, make clean).

I would prefer if you included both lkvm.mk and qemu.mk unconditionally, but instead modify the S1_RF_SECONDARY_STAMPS conditionally, like as follows:
```
UFK_INCLUDES := \
	kernel.mk \
	files.mk \
	lkvm.mk \
	qemu.mk
...
$(call inc-many,$(UFK_INCLUDES))

ifeq ($(KVM_HV_TAG),"hv_lkvm")
	S1_RF_SECONDARY_STAMPS += $(LKVM_STAMP)
else ifeq ($(KVM_HV_TAG),"hv_qemu")
	S1_RF_SECONDARY_STAMPS += $(QEMU_STAMP)
endif
```

For this to work, you should also exclude both `QEMU_STAMP` and `LKVM_STAMP` from the namespace undefine mechanism. You can do it like so:

in lkvm.mk at the bottom of the file:
```
$(call undefine-namespaces,LKVM,LKVM_STAMP)
```

in qemu.mk at the bottom of the file:
```
$(call undefine-namespaces,QEMU,QEMU_STAMP)
```

And remove the lines in lkvm.mk and qemu.mk modifying the `S1_RF_SECONDARY_STAMPS` variable.